### DISCUSSION TITLE
Claim Home

### DISCUSSION THREAD

**Discussion by MightyEclipseAWE** [GriefPrevention/GriefPrevention#1238] [Ideas]
An option to set a tp point for other players in claims. 
Default in the top middle of the claim.
/setclaimhome - to change the claim home position

**Comment by Jikoo** [comment]
Sounds like you might want [GPTeleport](https://github.com/Baktus79/GPTeleport). I don't believe there are any addons that allow _other_ users to teleport to your claims, but if you asked nicely the author might be able to add a config option to allow any user with a certain level of trust to tp.
